src/rates.ts

```diff
@@ -115,6 +115,9 @@ const addNewRatesToDocs = (
       documents[dbIndex] = {
         ...documents[dbIndex],
         ...rateMap,
+        ...bridgeCurrencies
+          .map(code => ({ [`${code}_${code}`]: '1' }))
+          .reduce((out, code) => ({ ...out, ...code }), {}),
```

Comment:
This can be done much faster in O(n) instead of O(n^2) like this:

`...bridgeCurrencies.reduce((out, code) => Object.assign(out, { [`${code}_${code}`]: '1' }), {})`